packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -349,9 +350,9 @@ export class OfficeAddinUsageData {
    * @param data Data object(s) sent to Application Insights
    */
   public reportExpectedException(method: string, err: Error | string, data: object = {}) {
-    let error = err instanceof Error ? err : new Error(`${this.options.projectName} error: ${err}`);
+    let error = err instanceof Error ? Object.create(err) : new Error(`${this.options.projectName} error: ${err}`);
     error.name = this.getEventName();
-    this.maskFilePaths(error);
+    error = this.maskFilePaths(error);
     const errorMessage = error instanceof Error ? error.message : error;
```

Comment:
If you've cloned it and we aren't cloning it inside maskFilePaths then you don't need to reassign it.